docs/contributing.md

```diff
@@ -179,3 +179,12 @@ To debug code generation the flag &#x2013;typescript.debugger can be used
 - Make sure to re-build autorest.typescript after any changes you want to test
 
       npm run build
+
+### I've got compliance violations for the generated SDKs. How do I resolve it?
```

Comment:
I would recommend having this section talk about dependency updates in general and not in the context of compliance violations. In the azure-sdk-for-js repo we now have automation looking for outdated dependencies and logging [issues with label dependency-upgrade-required](https://github.com/Azure/azure-sdk-for-js/issues?q=is%3Aopen+is%3Aissue+label%3Adependency-upgrade-required). If any of these are part of the generated code, then we will be opening issues in the autorest.typescript repo to update the dependency in the generated code. I would like to add help-wanted label on such issues in the code gen repo and point to the contributing guide as to how to go about making the change.

So, think of a section like `How to update dependencies in generated package.json files?".